lib/cdo/aws/cloud_formation.rb

```diff
@@ -206,7 +206,7 @@ def delete
       # supported by safe_load
       #
       # rubocop:disable Security/YAMLLoad
-      params = YAML.load(template)['Parameters']
+      params = YAML.load(template, permitted_classes: [Date])['Parameters']
```

Comment:
Psych is the new YAML parser, and its stricter by default. This fixes `rake adhoc:start`, see [related comment](https://github.com/code-dot-org/code-dot-org/pull/60329#issuecomment-2282220000)